src/games/gameitem.py

```diff
@@ -204,6 +196,14 @@ def update(self, message, client):
             client.usersUpdated.emit(self.players)
             return
 
+        # Map preview code
```

Comment:
Then move this back again where it's used, potentially inline it to make it less verbose?